sync/src/chain.rs

```diff
@@ -368,6 +369,25 @@ impl ChainSync {
 		}
 	}
 
+	/// @returns Information on peers connections
```

Comment:
rustdoc doesn't do anything with the `@` symbol like doxygen does, so it's probably unnecessary here.